capa/features/extractors/dnfile/insn.py

```diff
@@ -97,17 +98,32 @@ def extract_insn_api_features(fh: FunctionHandle, bh, ih: InsnHandle) -> Iterato
     if insn.opcode not in (OpCodes.Call, OpCodes.Callvirt, OpCodes.Jmp, OpCodes.Calli, OpCodes.Newobj):
         return
 
-    callee: Union[DnType, DnUnmanagedMethod, None] = get_callee(fh.ctx, insn.operand.value)
+    api_token: int
+
+    # ECMA: MethodSpec table records the signature of an instantiated generic method
+    #   Method (an index into the MethodDef or MemberRef table, specifying to which generic method this row refers)
+    #   Instantiation(an index into the Blob heap holding the signature of this instantiation)
+    if is_mdtable(insn.operand.table, "GenericMethod"):
+        # we map the generic method to its MethodDef or MemberRef parent
+        row: Optional[Any] = resolve_dotnet_token(fh.ctx["pe"], insn.operand)
+        if row is None:
+            return
+        # TODO Unknown1 == MethodSpec Method column; update pending https://github.com/malwarefrank/dnfile/issues/65
+        api_token = calculate_dotnet_token_value(row.Unknown1.table.number, row.Unknown1.row_index)
+    else:
+        api_token = insn.operand.value
+
+    callee: Union[DnType, DnUnmanagedMethod, None] = get_callee(fh.ctx, api_token)
     if callee is None:
         return
 
     if isinstance(callee, DnType):
         if callee.member.startswith(("get_", "set_")):
-            if insn.operand.table == METHODDEF_TABLE:
+            if is_mdtable(insn.operand.table, "MethodDef"):
```

Comment:
like comment in other PR, i'd prefer to see symbolic names versus strings, since strings can be mis-typed and also enums clearly document expected values, versus strings are unrestricted.